samples/entity-framework-core/SqlPersistence_7/Endpoint.SqlPersistence/Program.cs

```diff
@@ -11,11 +11,11 @@ class Program
     static async Task Main()
     {
         // for SqlExpress use Data Source=.\SqlExpress;Initial Catalog=NsbSamplesEfCoreUowSql;Integrated Security=True;Encrypt=false
-        var connectionString = @"Server=localhost,1433;Initial Catalog=NsbSamplesEfCoreUowSql;User Id=SA;Password=yourStrong(!)Password;Encrypt=false";
+        var connectionString = @"Server=localhost,1433;Initial Catalog=NsbSamplesEfCoreUowSql;User Id=SA;Password=yourStrong(!)Password;Encrypt=false;Max Pool Size=100";
```

Comment:
I always have to add: ` TrustServerCertificate=True`  should we add that as well?

I guess the bigger question is: should we standardize all our connection strings for samples since I assume we have plenty of them? (Might be yagni)

Reply:
This connection string and the other EF connection string are the same.  I added Max Pool Size because it throws a warning on the console if you don't specify it.  We haven't done the SQLT or P or NHibernate samples yet, It would be worth making sure they do the same things.  This connection string is setup to work with the docker container command in the sample markdown.